string first_name
string last_name
string teacher_name
